src/script/lua_api/l_object.cpp

```diff
@@ -907,13 +907,65 @@ int ObjectRef::l_get_yaw(lua_State *L)
 	NO_MAP_LOCK_REQUIRED;
 	ObjectRef *ref = checkobject(L, 1);
 	LuaEntitySAO *co = getluaobject(ref);
-	if (co == NULL) return 0;
-	// Do it
+	if (!co) return 0;
+
 	float yaw = co->getYaw() * core::DEGTORAD;
 	lua_pushnumber(L, yaw);
 	return 1;
 }
 
+// set_pitch(self, radians) (X-AXIS)
+int ObjectRef::l_set_pitch(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	LuaEntitySAO *co = getluaobject(ref);
+	if (!co) return 0;
```

Comment:
```
if (!co)
    return;
```
The action is always on a new line

Reply:
Again, I'm learning bad habits from old code, will fix it.

Reply:
See https://dev.minetest.net/Code_style_guidelines if you haven't already.

Reply:
Yeah, I looked at it and I've started following as well as I can.